importers/class-ucf-location-importer.php

```diff
@@ -407,6 +413,85 @@ private function update_terms( $post_id, $data ) {
 			);
 		}
 
+		/**
+		 * Adds orgs data to the location
+		 * @author Jim Barnes
+		 * @since 1.0.0
+		 * @param int $post_id The post ID
+		 * @param array $orgs The array of org data
+		 * @return void
+		 */
+		private function update_orgs( $post_id, $orgs ) {
+			// Start fresh with every import
+			$data = array();
+
+			foreach( $orgs as $i => $org ) {
+				$org_name = $org->name;
+				$org_phone = isset( $org->phone ) ? $org->phone : null;
+				$org_room = isset( $org->room ) ? $org->room : null;
+
+				$org_data = array(
+					'org_name'        => $this->smart_capitalize( $org_name ),
+					'org_phone'       => $org_phone,
+					'org_room'        => $org_room,
+					'org_departments' => array()
+				);
+
+				if ( isset( $org->departments ) && count( $org->departments ) > 0 ) {
+					foreach( $org->departments as $i => $dept ) {
```

Comment:
Probably doesn't matter since `$i` isn't used in either foreach loop, but there may be weirdness with `$i` being overwritten in this loop (previously defined on line 428)?

Again, not really a big deal since it's not utilized anywhere, but may need to be revisited if this code is ever changed to use `$i`.

Reply:
The `$i` iterators were there because initially I was using the `update_sub_field` function with the array selector: `update_sub_field( array( 'ucf_location_orgs', $i, 'sub_field_name'), $value, $post_id );`. I was only doing that because the `update_field` method wasn't working, but it was only not working because I have forgotten to include the `$post_id` in it. Just never removed the iterators after I switched back the other method.

TL;DR: I just removed the iterators.